recipes/flake8-nb/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "flake8-nb" %}
+{% set version = "0.1.1" %}
+{% set download_name = "flake8_nb" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ download_name }}/{{ download_name }}-{{ version }}.tar.gz"
+  sha256: "07cf86265cf8ddf17e438977b5717eaa77f8aa9e88b0fcca94c2e30f7e25fe2b"
+
+build:
+  number: 0
+  skip: true  # [py<36]
```

Comment:
This seems like it can be `noarch: python` if you add `python >=3.6` below and remove this line

Reply:
Now it somehow worked, while yesterday I failed all the time to set it up like this.
@scopatz Thanks for the review 😄